config/pubchem_pathway-ensembl_gene/config.yaml

```
@@ -0,0 +1,7 @@
+link:
+  forward: TIO_000068
+  reverse: TIO_000069
```

```suggestion
  forward: TIO_000084
  reverse: TIO_000085
```
gene は molecule でないので、TIO_000068 (pathway has participant molecule) やTIO_000069 (molecule participates in pathway) は使えないです。pathway has participant gene とかを後で新設しますので、まだないですがとりあえずこうしておきます。

---

承知しました